cypress/support/pages/home.pageObject.js

```diff
@@ -7,10 +7,51 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  get authorName() {
+    return cy.getByDataCy('author-name-home-page');
+  }
+
+  get followBtn() {
+    return cy.getByDataCy('follow-btn');
+  }
+
+  get unfollowBtn() {
+    return cy.getByDataCy('unfollow-btn');
+  }
+
+  clickOnProfileLink() {
+    this.usernameLink
+      .click();
+  }
+
+  clickOnAuthor() {
+    this.authorName
+      .click();
+  }
+
+  clickOnFollowBtn() {
+    this.followBtn
+      .click();
+  }
+
+  clickUnfollowBtn() {
+    this.unfollowBtn
+      .click();
+  }
+
   assertHeaderContainUsername(username) {
     this.usernameLink
       .should('contain', username);
   }
+
+  assertHeaderNotContainUsername(username) {
+    this.usernameLink.should('not.exist', username);
```

Comment:
The `assertHeaderNotContainUsername` method uses `should('not.exist', username)`, which is incorrect. Use `should('not.contain', username)` to verify that the username is not present.